src/hotkeys.js

```diff
@@ -76,7 +76,7 @@
           return false;
         }
 
-        return (element.contentEditable && element.contentEditable == 'true');
+        return (element.contentEditable && element.contentEditable == 'false');
```

Comment:
Won't this always return false?  What's the point in keeping this here at this point?  It seems as though the `stopCallback` should now just always return false